image/store/filesystem.go

```diff
@@ -172,6 +172,16 @@ func (ls LayerStorage) assembleTar(id LayerID) (io.ReadCloser, error) {
 	}), nil
 }
 
+func InitLayerDirectory() error {
+	dls := NewDefaultLayerStorage()
```

Comment:
I wonder if it is necessary to call this function here. We could just use layerDataRoot and layerDBRoot.